mock/etc/mock/site-defaults.cfg

```diff
@@ -205,6 +205,9 @@
 # config_opts['plugin_conf']['hw_info_enable'] = True
 # config_opts['plugin_conf']['hw_info_opts'] = {}
 #
+# config_opts['plugin_conf']['procenv_enable'] = True
```

Comment:
1) you should set the default in util.py as well (search for `hw_info_enable` there)
2) the information in procenv is too much detailed IMHO. I would propose to not enable it by default.

Reply:
Works for me.  I'll confess I really like the extra information it provides... but I can turn it on where I'm using it easily enough.